lib/telemetry_test.go

```diff
@@ -0,0 +1,9 @@
+package lib
+
+import (
+	"testing"
+)
+
+func TestInitTelemetry(t *testing.T) {
+	// TODO: add test cases for init telemetry sink
```

Comment:
It might make sense to just cover the case for the datadog config.

Reply:
Sounds good. Will add the test case.